src/components/themed/FioRequestRow.tsx

```diff
@@ -199,17 +200,20 @@ export const FioRequestRow = connect<StateProps, {}, OwnProps>(
         fiatAmount: ''
       }
     }
-    const tokenCode = fioRequest.content.token_code.toUpperCase()
+    let tokenCode = fioRequest.content.token_code.toUpperCase()
     try {
       const { allCurrencyInfos } = state.ui.settings.plugins
-      const plugin = allCurrencyInfos.find(plugin => {
-        const { currencyCode: pluginCurrencyCode } = plugin
+      const currencyInfo = allCurrencyInfos.find(info => {
```

Comment:
We have been trying to move away from `allCurrencyInfos`, since it's just a copy of what the core already has. Instead, please use `Object.keys(account.currencyConfig).find(pluginId => {...})` to directly iterate over the currencies in the account.

There are only two places left that still use `allCurrencyInfos`, so soon we can finally delete it.